octavia_f5/api/drivers/f5_driver/tasks/reschedule_tasks.py

```diff
@@ -121,3 +123,19 @@ def revert(self, result, load_balancer: models.LoadBalancer, candidate: str, rem
         LOG.warning("RewriteLoadBalancerEntry: Reverting host change of loadbalancer %s from '%s' to '%s'",
                     load_balancer.id, candidate, removal_host)
         self._loadbalancer_repo.update(db_apis.get_session(), load_balancer.id, server_group_id=removal_host)
+
+
+class SanityCheck(RescheduleTasks):
+    def execute(self, loadbalancer_id: str, candidate: str):
+        """Check the rescheduling parameters for sanity."""
+
+        # Check that target host exists.
+        # Else the LB would get deleted, but not recreated anywhere.
+        if not self._amphora_repo.get_devices_for_host(candidate):
```

Comment:
get_devices_for_host takes two parameters, not one: session and host.
I'd recommend you to add easy unit tests to your PRs.